hrjobcontract/CRM/Hrjobcontract/Dedupe/Merger.php

```diff
@@ -657,16 +712,11 @@ public static function getJoinOnDedupeTable() {
    *
    * @return string
    */
-  protected static function getWhereString($batchLimit, $isSelected) {
+  protected static function getWhereString($isSelected) {
```

Comment:
It's probably not caused by this PR specifically, but I notice that `$batchLimit` was removed from the arguments but not the doc-block.

Reply:
Good catch. I actually missed that one